runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/ProcessBundleDescriptors.java

```diff
@@ -190,12 +190,13 @@ public static ExecutableProcessBundleDescriptor of(
      * Get the {@link RemoteInputDestination} that input data are sent to the {@link
      * ProcessBundleDescriptor} over.
      */
-    public abstract RemoteInputDestination<WindowedValue<?>> getRemoteInputDestination();
+    public abstract RemoteInputDestination<? super WindowedValue<?>> getRemoteInputDestination();
 
     /**
      * Get all of the targets materialized by this {@link ExecutableProcessBundleDescriptor} and the
      * java {@link Coder} for the wire format of that {@link BeamFnApi.Target}.
      */
-    public abstract Map<BeamFnApi.Target, Coder<WindowedValue<?>>> getOutputTargetCoders();
+    public abstract Map<BeamFnApi.Target, Coder<? extends WindowedValue<?>>>
```

Comment:
I'm not sure we want to modify the coder type here.

Reply:
Done.

The `? super` means we don't have to do the double cast between `(Coder<WindowedValue<T>>) (Coder)`, which is why I'm keeping that one.

Reply:
Yes, I think we should keep the change for the sink-only data receiver.